vpr/src/base/ShowSetup.cpp

```diff
@@ -509,13 +509,22 @@ static void ShowPlacerOpts(const t_placer_opts& PlacerOpts,
             case RANDOM:
                 VTR_LOG("RANDOM\n");
                 break;
-            case USER:
-                VTR_LOG("USER '%s'\n", PlacerOpts.pad_loc_file.c_str());
-                break;
             default:
                 VPR_FATAL_ERROR(VPR_ERROR_UNKNOWN, "Unknown I/O pad location type\n");
         }
 
+        VTR_LOG("PlacerOpts.block_loc_type: ");
```

Comment:
Make more user friendly.
should just be PlacerOpts.constraint_file: fname

Reply:
At the same time, put a blank line before the first placer option (or after last packer option). Other groups have a blank line between them; placer and packer should too.